packages/astro/src/core/app/index.ts

```diff
@@ -41,6 +41,10 @@ export interface RenderErrorOptions {
 	routeData?: RouteData;
 	response?: Response;
 	status: 404 | 500;
+	/**
+	 * Whether to call onRequest() while rendering the error page. Defaults to true.
+	*/
+	runMiddleware?: boolean;
```

Comment:
nit: since this is the default behaviour is to always run the middleware, users that opt-in probably **don't** want to run the middleware. So maybe we can call it `bypassMiddleware` and by default is `false`

Reply:
Makes sense, woudn't take long.